vnext/ReactUWP/Views/SliderViewManager.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+
+#include <Views/ShadowNodeBase.h>
+#include "SliderViewManager.h"
```

Comment:
#include <Views/SliderViewManager.h> ?

Reply:
That's not how any of the other view managers include their header

---
In reply to: [335125441](https://github.com/microsoft/react-native-windows/pull/3400#discussion_r335125441) [](ancestors = 335125441)